cea/interfaces/dashboard/api/project.py

```diff
@@ -46,7 +48,7 @@ def get(self):
             config.scenario_name = ''
             config.save()
         return {'name': name, 'path': config.project, 'scenario': config.scenario_name,
-                'scenarios': config.get_parameter('general:scenario-name')._choices}
+                'scenarios': get_scenarios_from_project(config)}
```

Comment:
I like this! This could also be made into a Configuration method, but for now, let's keep it like this. OR: moved to the `cea.config` namespace? That would be even better!